api/nodes/views.py

```diff
@@ -40,7 +40,7 @@ def get_default_odm_query(self):
         return (
             Q('is_public', 'eq', True) &
             Q('is_deleted', 'ne', True) &
-            Q('is_dashboard', 'ne', True)
```

Comment:
I think we Dashboard and folders should not be returned for the /nodes/ endpoint.